Packs/Armorblox/ReleaseNotes/1_0_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
```

Comment:
Please remove all the unnecessary spaces in the release notes files

Reply:
I looked at the rest of RL as well and removed all the unnecessary spaces + added docker updated when it was updated